server/game/core/ability/PlayerOrCardAbility.js

```diff
@@ -59,9 +60,15 @@ class PlayerOrCardAbility {
         this.uuid = uuidv4();
 
         // TODO: Ensure that nested abilities(triggers resolving during a trigger resolution) are resolving as expected.
-        this.resolveTriggersAfter = properties.resolveTriggersAfter != null
-            ? properties.resolveTriggersAfter
-            : [AbilityType.Triggered, AbilityType.Action].includes(this.type);
+        this.triggerHandlingMode = properties.triggerHandlingMode;
+
+        if (properties.triggerHandlingMode != null) {
```

Comment:
Just making sure you know this is `!=` instead of `!==`. I did a bit of research but I'm still not sure which is more appropriate.

Reply:
The practice that was recommended to me by a TS dev is to use `!=` and `==` when checking null or undefined since they are sometimes used interchangeably